docs/Rules/ReservedParams.md

```diff
@@ -11,7 +11,7 @@ title: ReservedParams
 
 ## Description
 
-You cannot use reserved common parameters in an advanced function.
+You cannot use [reserved common parameters](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_commonparameters) in an advanced function.
```

Comment:
Suggest to remove language/culture, browser will automatically add correct one based on user settings
```suggestion
You cannot use [reserved common parameters](https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_commonparameters) in an advanced function.
```